dev/src/bulk-writer.ts

```diff
@@ -403,10 +409,10 @@ export class BulkWriter {
   /**
    * Commits all writes that have been enqueued up to this point in parallel.
    *
-   * Returns a Promise that resolves when there are no more pending writes. It
-   * never fails.
+   * Returns a Promise that resolves when there are no more pending writes.
```

Comment:
Maybe:

"Returns a Promise that resolves when all currently queued operations have been committed. The Promise returned from this method will never be rejected as the success for each individual operation is conveyed via their individual Promises. "

We do need to rewrite the "when there are no more pending writes" part, the rest I am on the fence about.

Reply:
"Returns a Promise that resolves when all currently queued operations have been committed. The Promise will never be rejected since the results for each individual operation are conveyed via their individual Promises."

Renamed return to: `@return {Promise<void>} A promise that resolves when all enqueued writes up to this point have been committed.`